fix_authors_in_json.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# fix errors in database
```

Comment:
What kind of errors? What is changed?

My python knowledge is limited, but it looks like this is printing items that need to be fixed manually, not changing them?